Readme.md

```diff
@@ -29,7 +35,13 @@ The full documentation of Tracee is available at
 You can use the version selector on top to view documentation for a specific
 version of Tracee.
 
-## Quickstart
+## Quickstart (Kubernetes)
+
+Tracee is designed to monitor hosts in kubernetes clusters. To see this in action check out the quickstart [here](./docs/getting-started/kubernetes-quickstart.md).
+
+## Quickstart (docker)
```

Comment:
Would you want to link here to the Docker starter-guide/quickstart similar to how you link to the Kubernetes quickstart above?